src/sentry/web/frontend/js_sdk_loader.py

```diff
@@ -171,6 +188,17 @@ def get(self, request: Request, public_key: Optional[str], minified: Optional[st
 
         metrics.incr("js-sdk-loader.rendered", instance=instance, skip_internal=False)
 
+        analytics.record(
+            "js_sdk_loader.rendered",
+            org_id=key.project.organization_id,
+            is_lazy=loader_config["isLazy"],
+            has_performance=loader_config["hasPerformance"],
+            has_replay=loader_config["hasReplay"],
+            has_debug=loader_config["hasDebug"],
+            sdk_version=sdk_version,
+            tmpl_instance=instance,
```

Comment:
m: Can we just store `tmpl` instead (`sentry/js-sdk-loader-noop.js.tmpl`)? Feel like that is easier to understand than default/noop/minified.

Reply:
sure thing!